software/nosql/src/main/java/brooklyn/entity/nosql/riak/RiakNodeSshDriver.java

```diff
@@ -251,6 +251,15 @@ public void customize() {
             String saveAsRiakConf = Urls.mergePaths(getRunDir(), "riak.conf");
             DynamicTasks.queueIfPossible(SshEffectorTasks.put(saveAsRiakConf).contents(riakConfTemplate));
             commands.add(sudo("mv " + saveAsRiakConf + " " + getRiakEtcDir()));
+
+            if(!osDetails.isMac()) {
```

Comment:
I see that there is RIAK_CONF_TEMPLATE_URL_MAC and RIAK_CONF_TEMPLATE_URL_LINUX .
I am not familiar with differences between Riak conf on Mac and Linux but I guess your are right and we can use RIAK_CONFIG_FILE for both.